scripts/test_real_world_ingestion.py

```
@@ -0,0 +1,885 @@
+            headers=headers_with_key,
+            json=event
+        )
+        
+        response2 = await self.client.post(
+            f"{BASE_URL}/api/v1/ingestion/events",
+            headers=headers_with_key,
+            json=event
+        )
+        
+        if response1.json().get('event_id') == response2.json().get('event_id'):
+            print(f" Idempotency maintained: same event returned")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)